src/data/markdown/docs/05 Examples/01 Examples/02 http-authentication.md

```diff
@@ -108,93 +108,68 @@ export default function () {
 
 </CodeGroup>
 
-## AWS Signature v4 authentication
+## AWS Signature v4 authentication with the [k6-jslib-aws](https://github.com/grafana/k6-jslib-aws)
 
-Requests to the AWS APIs requires a special type of auth, called AWS Signature Version 4. k6
-does not support this authentication mechanism out of the box, so we'll have to resort to using
-a Node.js library called [awsv4.js](https://github.com/mhart/aws4) and
-[Browserify](http://browserify.org/) (to make it work in k6).
+To authenticate requests to AWS APIs using [AWS Signature Version 4](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html), k6 offers the [k6-jslib-aws](https://github.com/grafana/k6-jslib-aws) javascript library which provides a dedicated `SignatureV4` class. This class allows to produce authenticated requests that can be sent to AWS APIs using the `http` k6 module.
```

Comment:
```suggestion
To authenticate requests to AWS APIs using [AWS Signature Version 4](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html), k6 offers the [k6-jslib-aws](https://github.com/grafana/k6-jslib-aws) JavaScript library, which provides a dedicated `SignatureV4` class. This class can produce authenticated requests to send to AWS APIs using the `http` k6 module.
```

Reply:
In conventional style, "which" is set off by a comma and that isn't. For a deeper dive into the reasons why, you can look at restrictive vs unrestrictive clauses:

https://owl.purdue.edu/owl/general_writing/grammar/that_vs_which.html

Reply:
TIL, thanks a lot for the learning 🙇🏻